src/utils/mappers.js

```diff
@@ -15,5 +15,9 @@ export function req ( path ) {
 }
 
 export function globalify ( name ) {
-	return 'global.' + name;
-}
\ No newline at end of file
+  	if ( /^__dep\d+__$/.test( name ) ) {
```

Comment:
I hear you. I think this is fine for now - if we ever revisit how these names are constructed (e.g. to allow for different global v local names, e.g. `global.jQuery` but `(function ($) {...`, then the refactor will happen at that point. Don't think there's any harm in the time being